Sources/MongoSwift/MongoClient.swift

```diff
@@ -1,7 +1,9 @@
 import Foundation
 import mongoc
 
-/// Options to use when creating a `MongoClient`.
+/// Options to use when creating a `MongoClient`. For options that are also included in the connection string URI, the
```

Comment:
IMO, the second sentence on maybe makes more sense to include in the docstring for the `MongoClient` initializer, so it's clear what the connection string you're referring to is.